web3/Dockerfile

```
@@ -1,9 +1,9 @@
 
 FROM node:14-bullseye
 
-COPY . /godwoken/.
+COPY ./web3 /godwoken-web3
 
-WORKDIR /godwoken/web3
+WORKDIR /godwoken-web3
 
 RUN yarn && yarn build
```

```suggestion
RUN cd /godwoken-web3 && yarn && yarn build
```
see https://github.com/godwokenrises/godwoken/commit/1a3174548febd199a44ccd331ba4d64299676014#diff-6ec036556353d556083fa7858cef52cfece85508440e56c79c9c9d3476b96a35L5